app/src/test/java/com/x8bit/bitwarden/ui/vault/feature/importlogins/ImportLoginsScreenTest.kt

```diff
@@ -323,17 +321,11 @@ class ImportLoginsScreenTest : BaseComposeTest() {
         verifyActionSent(ImportLoginsAction.MoveToStepTwo)
     }
 
-    @Test
-    fun `NavigateToImportSuccess event causes correct lambda to invoke`() {
-        mutableImportLoginsEventFlow.tryEmit(ImportLoginsEvent.NavigateToImportSuccess)
-        assertTrue(navigateToImportLoginSuccessCalled)
-    }
-
     @Test
     fun `Loading content is displayed when isVaultSyncing is true`() {
-        mutableImportLoginsStateFlow.update {
-            it.copy(isVaultSyncing = true)
-        }
+        mutableImportLoginsStateFlow.tryEmit(
```

Comment:
Can we leave these as `update {`

Reply:
I will switch them all if that works? I moved this to tryEmit to match the rest...

Reply:
Yea, we should just use Update